src/client/components/InstructionsComponent/InstructionsComponent.css

```diff
@@ -0,0 +1,30 @@
+.container{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: fit-content;
+    
+}
+.all-images{
+    display: flex;
+    margin-top: 30px;
+    padding: 50px;
+    
+    
```

Comment:
We might have some bottom padding to add more white spaces like the design we have in the figma.